spec/spec.md

```diff
@@ -80,21 +80,27 @@ DELETE <ECHO_SERVER_URL>/clients/<CLIENT_ID>
 
 ### Send Notification
 
-> **Note**
-> This will have E2E encryption in the future but for now is designed without encryption.
-
 ```
 POST <ECHO_SERVER_URL>/clients/<CLIENT_ID>
 {
     "id": "0000-0000-0000-0000"
     "payload": {
-        "title": "New Request",
-        "description": "You have a signing request."
+      "topic": "<topic>",
+      "flags": <flags>,
+      "blob": "<blob>"
     }
-}
 ```
 
 - `CLIENT_ID`: This is used to map to the device token
 - `id`: This is generated by the Relay to prevent duplicate notifications
-- `payload`: A JSON object that holds the relevant payload data. **NOTE:** This will become a string that is encrypted
-  by the dApp and not decrypted until received by the client
\ No newline at end of file
+- `payload`: A string with 3 values encoded and split with a `:` as a delimiter.
+  - `topic`: This is used by the SDKs to decrypt encrypted payloads on the client side
+  - `flags`: an integer value that uses bitmasks to encode information about the payload for the SDKs, the flags are below
```

Comment:
Let's make sure we stay within 1 byte if we use bit masks to avoid endianness issues.

Reply:
👍🏻 Might remove it if we decide it's unneeded